content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,302 @@
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is set` in the next field.
+
+Next, click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is set` to
+create a new line of fields beneath it.
+
+Set the first field to :guilabel:`Active`. Then, select :guilabel:`is not set` in the next field.
+
+.. image:: lead_distribution_report/active-set.png
+  :align: center
+  :alt: Use Active to include active status in the report.
+
+This rule adds the activity status of the lead to the report.
+
+.. note::
+  Active status is an important filter to include when creating a lead distribution report because
+  it includes **all** leads regardless of won/lost or active/inactive status in the report. This
+  provides a comprehensive view of all the leads assigned to each sales member.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`essential-filters` in the order specified to achieve a heavily-detailed filter.
+
+:ref:`referred-by`
+
+:ref:`source`
+
+:ref:`notes`
+
+:ref:`tags`
+
+:ref:`email`
+
+:ref:`salesperson`
+
+These conditions can be added, removed, or modified to best fit the desired information in the
+report.
+
+.. _referred-by:
+
+1. Referred by
+~~~~~~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to :guilabel:`Active is not set` to
+create a new line of fields beneath it.
+
+Next, set the first field to :guilabel:`Referred By`. In the second field, select
+:guilabel:`contains` from the drop-down menu. In the last field, type `appointment`.
+
+.. image:: lead_distribution_report/referred-by.png
+  :align: center
+  :alt: Add referred by as a branch to filter appointments.
+
+This rule adds any leads that were referred by an appointment to the report.
+
+.. _source:
+
+2. Source
+~~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to the line for `Referred by
+appointment`.
+
+Next, set the first field to :guilabel:`Source`. Select :guilabel:`contains` from the drop-down menu
+in the second field. Type `livechat` in the third field.
+
+This rule adds any leads that came from livechat to the report.
+
+.. _notes:
+
+3. Notes
+~~~~~~~~
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` button next to line for `Source contains livechat`.
+
+Next, set the first field to :guilabel:`Notes`. Click the second field, and select
+:guilabel:`contains` from the drop-down menu. Type `mrp` in the third field.
+
+Click the :icon:`fa-plus` :guilabel:`(New Rule)` icon next to the line for `Notes contains mrp`, and
+add a new rule for notes containing `stock`. Repeat the process to create new rules containing the
+following terms:
+
+#. purchase
+#. plm
+#. crm
+#. sales
+#. project
+#. fsm
```

Comment:
Pretty good example where we're being too specific with the values instead of sticking to the parameter (**Notes** in this case). These particular values aren't useful to any other database except ours because they are coded into the apps people choose when they sign up for a free trial ^_^